docs/config.rst

```diff
@@ -1,11 +1,502 @@
+The following would replace the fourth step of the ``dev_org`` flow with a custom task that loads data into a dev environment.
+
+.. code-block:: yaml
+
+    dev_org:
+        steps:
+            4:
+                task: load_data_dev
+
+You can swap two steps in a flow by replacing one with the other. 
+
+
+
+
+
+
+Configuring Options on Tasks When Running a Subflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+TODO
+
+
+
+Using ``when`` Clauses
+^^^^^^^^^^^^^^^^^^^^^^
+TODO
+
+
+Org Configurations
+-------------------
+This section shows you the values that are overridable in an org configuration via the `cumulusci.yml` file.
+If you're looking for more information using scratch orgs with CumulusCI, see `Scratch Org Environments`_. 
+
+Override a Default Values
+^^^^^^^^^^^^^^^^^^^^^^^^^
+These overrides pertain only to scratch orgs.
+You can override the following values for org:
+
+* ``days`` - integer - Number of days for the scratch org to persist.
+* ``namespaced`` - boolean - Whether or not the scratch org is a `namespaced org <TODO sf link?>`_.
+* ``config_file`` - string - Path to the org definition file to use when building the scratch org.
+
+To override a value you can use the following syntax
+
+.. code-block:: yaml
+    
+    orgs:
+        org_name:
+            key: value 
+
+The following demonstrates how to override the default number of days from 7 to 15 on the ``dev`` org:
+
+.. code-block:: yaml
+
+    orgs:
+        dev:
+            days: 15
+
+
+Configuration Scopes
+--------------------
+When we reference ``cumulusci.yml`` in our documentation, we are referring to the ``cumulusci.yml`` file located in your project's root directory.
+In actuality, CumulusCI merges multiple `YAML <https://yaml.org/>`_ files that allow for configuration to occur at several distinct scopes.
+All of these files are have the same name- ``cumulusci.yml`` -but live in different locations on the file system.
+
+You can configure files at three scope levels: Project, Local Project, Global.
+Configurations have the following order of override precedence (from highest to lowest):
+
+* Local Project
+* Project
+* Global
+
+One override will only cascade over another when two configurations are setting a value for the same element on a task or flow.
+For example, say there exists a task T that takes two options o1 and o2.
+You can specify a default value for o1 in your project's ``cumulusci.yml`` file and a default for o2 in your global ``cumulusci.yml`` file and everything will work as exepected.
+If you then change your project's ``cumulusci.yml`` file to also specify a default value for o2, this will take precedence over the default value specified in your global ``cumulusci.yml`` file.
+
+The following diagram illustrates these three files along with their corresponding scopes in green.
+Changes made to configuration files on top will override any changes in files below them (if they are present).
+
+.. image:: images/cci-yml-chart.png
```

Comment:
The diagram is nice, but needs to be updated to use the preferred terms.

What tool was used to create the diagram? Is there a source file that we can include in the doc repository to help the next person who needs to update it?

Also consider that images which include text are less accessible. We should either avoid the image or include the same content as alt text for the image, if possible.